dune/grid/cpgrid/GridHelpers.cpp

```diff
@@ -30,6 +31,30 @@ namespace Opm
 namespace UgGridHelpers
 {
 
+EclipseGrid createEclipseGrid(const Dune::CpGrid& grid, const EclipseGrid& inputGrid)
+{
+    const int * dims = cartDims( grid );
+    if ((inputGrid.getNX( ) == static_cast<size_t>(dims[0])) &&
+        (inputGrid.getNY( ) == static_cast<size_t>(dims[1])) &&
+        (inputGrid.getNZ( ) == static_cast<size_t>(dims[2]))) {
+
+        std::vector<int> updatedACTNUM;
+        const int* global_cell = globalCell( grid );
+        updatedACTNUM.assign( inputGrid.getCartesianSize( ) , 0 );
```

Comment:
Just for reference, `std::vector` has a `vector< T >( size_t, T )` constructor that does construct + assign.